webpack/assets/javascripts/react_app/components/FactCharts/FactChartActions.js

```diff
@@ -1,23 +1,15 @@
 import {
-  FACT_CHART_REQUEST,
-  FACT_CHART_SUCCESS,
-  FACT_CHART_FAILURE,
+  FACT_CHART,
   FACT_CHART_MODAL_OPEN,
   FACT_CHART_MODAL_CLOSE,
 } from './FactChartConstants';
 
-import { ajaxRequestAction } from '../../redux/actions/common/index';
-
-export const getChartData = (url, id) => dispatch =>
-  ajaxRequestAction({
-    dispatch,
-    requestAction: FACT_CHART_REQUEST,
-    successAction: FACT_CHART_SUCCESS,
-    failedAction: FACT_CHART_FAILURE,
-    url,
-    item: { id },
-  });
-
+export const getChartData = (url, id) => ({
+  type: 'API_GET',
```

Comment:
I think `apiRequestType` or something similar would make more sense,
and the argument should be one that our `API` object should be familiar with, for example:
`get, put, post, delete, patch`

Reply:
I want to suggest using consts like `API_OPERATIONS.GET`.

Another approach is creating helper functions so you would use them as
```js
export const getChartData = (url, id) => createGetAction({ url, id, subType, ... });
```

Reply:
Was just adding them :)
I think a helper function will be unnecessary because it looks just like an extra step